sdk/src/main/java/com/mapbox/maps/Snapshotter.kt

```diff
@@ -425,25 +434,19 @@ class Snapshotter : MapSnapshotterObserver {
     val scale: Float
   )
 
-  /**
-   * Callback for getting snapshot when it's finished.
-   */
-  interface SnapshotReadyCallback {
-    /**
-     * @param mapStyleDelegate The style delegate
-     */
-    fun onStyleLoaded(style: Style)
-
-    /**
-     * @param snapshot An image snapshot of a map rendered by MapSnapshotter.
-     */
-    fun onSnapshotCreated(snapshot: Expected<MapSnapshotInterface?, String?>)
-  }
-
   /**
    * Static variables and methods.
    */
   companion object {
     private const val TAG = "Snapshotter"
   }
+}
+
+fun MapSnapshotInterface.bitmap(): Bitmap {
```

Comment:
nit documentation

Reply:
I would suggest to also add an extension function to `Image` class and convert it to bitmap, the extension function can be located at sdk-base.

Reply:
I see no sense in adding extension function for `Image` while it is not used.
However moved `MapSnapshotInterface.bitmap()` to `sdk-base`, that sounds good.